plugins/imfile/imfile.c

```diff
@@ -2042,6 +2073,7 @@ CODESTARTbeginCnfLoad
 	cs.readMode = 0;
 	cs.maxLinesAtOnce = 10240;
 	cs.trimLineOverBytes = 0;
+	cs.ignoreOlderThan = 0;
```

Comment:
please remove support for the old config system - we do NOT provide new features via the old system.